extra-wm/i3-gaps/autobuild/beyond

```diff
@@ -0,0 +1,3 @@
+mkdir -p "$PKGDIR"/usr/share/man/man1/
```

Comment:
Just as a stylistic recommendation, use verbose commands to help tracking the situation, or at least add a `abinfo "What is going on here..."` to indicate what is being done here.

Reference: [AOSC OS Package Styling Manual](https://wiki.aosc.io/developers/aosc-os-package-styling-manual)